config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
I feel like we don't do a good enough job of explaining how to get these pieces. Do they need to set up the app settings from their github fork?